### PR TITLE
Updated w3bdesign and fixed broken image

### PR DESCRIPTION
**Pull request by w3bdesign**
As the profile has been modified slightly recently, I have attached updated screenshots.

Also fixed a broken screenshot of Savio Martin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,5 +26,5 @@ Any questions let us know!
 
 <!-- DO NOT EDIT -->
```
Comment (AllanRegush) [inline README.md:27]: Anything in these comments `<!--START_SECTION:data-section-->` `<!--START_SECTION:data-section-->`  will get overridden by our [GitHub Action](https://github.com/EddieJaoudeCommunity/gh-action-community). I recommend that you revert these changes as any changes not expected will be removed.

**Review by AllanRegush — COMMENTED** [review]
Thank for your PR 👍 Looking Good

File: README.md (hunk 1 above)
Comment (w3bdesign) [reply README.md:27]: I made sure to only modify the image URL in data.json and README.md, everything else should be unmodified.

**Review by w3bdesign — COMMENTED** [review]
I made sure to only modify the image URL in data.json and README.md, everything else should be unmodified.
